Add error bar to bar plot

Scemp opened the pull request:
This PR introduces error bars to bar plot when importing the data.

hdsassnick commented:
@Scemp, thanks for the contribution. On first sight, I am just a bit puzzled why the tests needed to be changed?

Scemp commented:
Hi @hdsassnick ,
While working on the PR, I noticed that 'y_values' are not needed to import bar data. Thus, I removed the parameter and since it was necessary before, the test failed. I adjusted the test accordingly and removed 'y_values' from the input parameters.
